src/main/java/seedu/address/model/Model.java

```diff
@@ -5,14 +5,14 @@
 
 import javafx.collections.ObservableList;
 import seedu.address.commons.core.GuiSettings;
-import seedu.address.model.person.Person;
+import seedu.address.model.task.Task;
 
 /**
  * The API of the Model component.
  */
 public interface Model {
     /** {@code Predicate} that always evaluate to true */
-    Predicate<Person> PREDICATE_SHOW_ALL_PERSONS = unused -> true;
+    Predicate<Task> PREDICATE_SHOW_ALL_PERSONS = unused -> true;
```

Comment:
Should `PREDICATE_SHOW_ALL_PERSONS` be renamed as `PREDICATE_SHOW_ALL_TASKS` as well?

Reply:
Yup it should be, thanks for pointing that out!